README.md

```diff
@@ -343,6 +343,7 @@ api to construct a junit report.  This is useful to track tests in CI such as Je
 - `--import-variable-csv-file /path/to/csv/file.csv` - Use with `run` and `--import-variable-name` to upload new tabular variable values before your run to specify the path to your CSV file.
 - `--import-variable-name NAME` - Use with `run` and `--import-variable-csv-file` to upload new tabular variable values before your run to specify the name of your tabular variable. You may also use this with the `csv-upload` command to update your variable without starting a run.
 - `--single-use` - Use with `run` or `csv-upload` to flag your variable upload as `single-use`. See `--import-variable-csv-file` and `--import-variable-name` options as well.
+- `--max-retries` - If set to a value > 0 and a test fails, the CLI will re-run failed tests a number of times before reporting failure. Cannot be used together with `--fail-fast`.
```

Comment:
Could you document what happens if you use this with the `--junit-file` parameter?